src/web_app/src/components/log_row.tsx

```diff
@@ -11,15 +12,25 @@ import Typography from "@mui/material/Typography";
 import Table from "@mui/material/Table";
 import TableHead from "@mui/material/TableHead";
 import TableBody from "@mui/material/TableBody";
+import PlayCircleFilledIcon from "@mui/icons-material/PlayCircleFilled";
+import Button from "@mui/material/Button";
+import {useLocation, useNavigate} from "react-router-dom";
 
 
 export default function Row(props: { row: Types.Log }) {
     const {row} = props;
+    const url = useLocation().pathname;
```

Comment:
This line makes a test fails. The test is the `log_page.test.tsx` and among the 400 pages jest has produced to explain to me that the test failed, I found this line `useLocation() may be used only in the context of a <Router> component.` I hope it will be useful. If you need the complete logs, I can send them to you.

Reply:
I fixed it myself.